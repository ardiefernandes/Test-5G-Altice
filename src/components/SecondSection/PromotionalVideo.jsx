import React from "react";

const PromotionalVideo = () => {
  return (
    <div className="second-section-video">
      <video
        alt="video promocional top gun maverick na meo"
        autoPlay
        muted
        loop
        src="https://www.meo.pt/PublishingImages/videos/top-gun-maverick-meo-video-xl/top-gun-maverick-meo-video-xl.mp4"
      ></video>
    </div>
  );
};

export default PromotionalVideo;
