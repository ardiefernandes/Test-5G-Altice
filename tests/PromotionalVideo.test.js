import React from "react";
import renderer from "react-test-renderer";
import PromotionalVideo from "../src/components/SecondSection/PromotionalVideo";

it("correctly renders the second section promotional video", () => {
  const component = renderer.create(<PromotionalVideo />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
