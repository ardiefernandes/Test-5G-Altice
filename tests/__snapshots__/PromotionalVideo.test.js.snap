// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders the second section promotional video 1`] = `
<div
  className="second-section-video"
>
  <video
    alt="video promocional top gun maverick na meo"
    autoPlay={true}
    loop={true}
    muted={true}
    src="https://www.meo.pt/PublishingImages/videos/top-gun-maverick-meo-video-xl/top-gun-maverick-meo-video-xl.mp4"
  />
</div>
`;
